### DISCUSSION TITLE
Positive feedback 🌟

### DISCUSSION THREAD

**Discussion by dminca** [glanceapp/glance#159] [General]
Hi, just wanted to say that I gave the app a shot today in my test Kubernetes cluster, also created a Helm Chart for it and I absolutely fell in love with it ❤️ 

Thank you so much for putting it out here and for the awesome features you've built into it!

If you need the Helm Chart, I can contribute with the source code if it helps other Kubernetes people out there 😄 

Cheers 🥂

**Comment by danz0l99** [comment]
I, like you, fell in love with it and its fair more versatile than say homepage.

I would suggest sharing your Helm Chart widget as the more widget ideas with code, the better it is for everyone and builds on the popularity and user instances of this excellent homepage replacement.

**Comment by svilenmarkov** [comment]
Thanks for the kind words!

As for the helm chart, I personally don't use k8s but if it's something that might be of help to others I don't see why not.
